packages/astro/src/vite-plugin-css/util.ts

```diff
@@ -0,0 +1,11 @@
+import { getVirtualModulePageName } from '../vite-plugin-pages/util.js';
+
+const MODULE_DEV_CSS_PREFIX = 'virtual:astro:dev-css:';
+
+/**
+ * Get the virtual module name for a dev CSS import.
+ * Usage: `await loader.import(getDevCSSModuleName(routeData.component))`
+ */
+export function getDevCSSModuleName(componentPath: string): string {
+	return getVirtualModulePageName(MODULE_DEV_CSS_PREFIX, componentPath);
+}
```

Comment:
I think this utility should be moved next to its opposite function. Plus, we're repeating the module prefix, while we should instead use one constant to reduce errors